src/ui/TaskRenderer.svelte

```diff
@@ -101,6 +102,11 @@
     <div bind:this={taskContentEl} class="todoist-task-content" />
   </div>
   <div class="task-metadata">
+    {#if settings.renderDescription && todo.description}
+      <div class="task-description">
+        <SvelteMarkdown source={todo.description} />
+      </div>
```

Comment:
We should render this Markdown the same way we do for the task content. See `renderMarkdown` in this file for an example.

This goes through Obsidian's renderer which will handle things like page links/references correctly. Also means one less dependency!